src/resources/partners.md

```diff
@@ -85,3 +85,45 @@ partners_list:
                 class: youtube
                 url: 'https://www.youtube.com/moduscreate'
 ---
+        name: Monterail
+        logo: Monterail.png
```

Comment:
We need an image to be included in the PR, under this directory: https://github.com/vuejs/vuejs.org/tree/master/themes/vue/source/images